src/pages/matching/index.tsx

```diff
@@ -14,6 +14,25 @@ import { ScreenReaderOnly } from 'styles';
 const MatchingRule: NextPage = () => {
   const router = useRouter();
 
+  const RULE_LIST = [
+    {
+      icon: <EnIcon />,
+      text: '한국어 보다는 영어 사용을 권장합니다.',
+    },
+    {
+      icon: <BlockIcon />,
+      text: '특정 횟수의 경고를 받은 사용자는 차단될 수 있습니다.',
+    },
+    {
+      icon: <PrivacyIcon />,
+      text: '동의하지 않은 개인정보를 노출되거나 요청을 금지합니다.',
+    },
+    {
+      icon: <DiseaseIcon />,
+      text: '선정적인 언어, 혐오/차별/폭력적인 언어, 불법 행위/기타 행위를 제한합니다.',
+    },
+  ];
```

Comment:
p3;
목록에 id 값도 추가하여 key 값에 index 대신 사용하면 좋을 것 같아요!

```suggestion
  const RULE_LIST = [
    {
      id: 1,
      icon: <EnIcon />,
      text: '한국어 보다는 영어 사용을 권장합니다.',
    },
    {
      id: 2,    
      icon: <BlockIcon />,
      text: '특정 횟수의 경고를 받은 사용자는 차단될 수 있습니다.',
    },
    {
      id: 3,
      icon: <PrivacyIcon />,
      text: '동의하지 않은 개인정보를 노출되거나 요청을 금지합니다.',
    },
    {
      id: 4,
      icon: <DiseaseIcon />,
      text: '선정적인 언어, 혐오/차별/폭력적인 언어, 불법 행위/기타 행위를 제한합니다.',
    },
  ];
```

Reply:
아이디 추가했습니다!

작업 커밋: 7787269